repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -243,6 +251,20 @@ private void initNotifiers() {
         this.notifiers.add(new FixerBuildNotifier(notifierEngines));
     }
 
+    private List<String> getListOfProjectsToIgnore() {
+        List<String> result = new ArrayList<>();
+        if (this.config.getProjectsToIgnoreFilePath() != null) {
+            try {
+                for (String line : Files.readAllLines(new File(this.config.getProjectsToIgnoreFilePath()).toPath())) {
```

Comment:
Please use a local variable for this `Files.readAllLines(new File(this.config.getProjectsToIgnoreFilePath()).toPath())`: it's more readable and can be useful in case of bug

Reply:
Done,